### ISSUE TITLE
Authentication Header Bearer issue

### ISSUE THREAD

**Issue by igortsev** [n8n-io/n8n#9147]
### Bug Description

Some issues with header Authentication. i have access_token with 227 letter lenght and then try to send request access_token not sent or was trim. When i do without using Generic Credentials -> Credential for Header Auth and just set Send Headers with the same name Authorization and Value Bearer some_token_227_letters

### To Reproduce

1. Create any POST request with Generic Credentials -> Credential for Header Auth
2. Set name Authentication and Value as : Bearer some_token_lenght_230_letters
3. Filed header auth -> "User authorization failed: no access_token passed.",

### Expected behavior

Success auth

### Operating System

Ubuntu 22.04

### n8n Version

1.36.4

### Node.js Version

last

### Database

PostgreSQL

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @igortsev,

Can you provide a workflow that reproduces this? I have generated a 230 character string sent it as a header to webhook.site and I can see the value is there and not edited.

I have also sent the same value as as a GET request to httpbin.org/headers which returns the headers and I can see it is set there as well.

**Comment by igortsev** [comment]
As example https://dev.vk.com/en/api/market/overview that API require header authorization via access_token. Access token should be send `Autorization: Bearer access_token` . I made POST/GET request via Postman and request was success with token. I made the same with N8N.

What i did ?

1. Select Authentication -> Generic Auth Type -> Header Auth
2. Setup credentials -> Name Autorization and value Bearer access_token
3. Got error -> User authorization failed: no access_token passed
4. Console log display HttpRequest object where header Autorization exist with value : hidden

I did the same but without Select Authentication -> Generic Auth Type -> Header Auth and just manual enable "Send headers" 

- KEY Authentication 
- Value Bearer access_token
and its works fine.

The same time i got another issue with HttpRequest, i just try to get XML file with data via GET request. Postman was success, one more N8N instalation was success too, nut my own via Docker just hold and not provide any response. I used culr via SSH and was success with few seconds to get it.
